app/models/docker_builder_service.rb

```diff
@@ -82,7 +82,10 @@ def run(push: false, tag_as_latest: false)
         if build_image(tmp_dir) # rubocop:disable Style/IfInsideElse
           ret = true
           ret = push_image(tag_as_latest: tag_as_latest) if push
-          build.docker_image.remove(force: true) unless ENV["DOCKER_KEEP_BUILT_IMGS"] == "1"
+          unless ENV["DOCKER_KEEP_BUILT_IMGS"] == "1"
+            output.puts("Deleting docker image #{build.docker_image.id}")
```

Comment:
this returns the `sha256:41d6fa09472fdf8ed663cb36d0dae276b632729c7df6a3693f2e86266acfbab0` ... which is a bit ugly ... should return the short-form id instead ? (same as shown in `docker images`)

```
id = build.docker_image.id.to_s
id = id[/sha256:(.{12})/, 1] || id
```